MAINTAINERS.md

```diff
@@ -1,18 +1,145 @@
+
+There's no fixed number of PRs every maintainer needs to look at to be considered "active", but if you have to ask "am I inactive?" that's probably something you should worry about.
+
+2.  Triaging open issues in the community.
+
+The expectation is that you're spending a portion of your day looking at issues.  If we have one a few issues coming in every day, you can do it over your morning coffee.  When you're triaging, you should:
+	*Add "help wanted" labels to good issues for new community members.
+	*Add "Severity-Blocking" labels if an issue scares you.  The community doesn't have a good way to page us right now, so if you see a blocking bug, mark it as Blocking and engage oncall.
+	*Request for more information from a submitter if an issue is not clear.
+
+3. Responding to enhancement requests and forum posts
+
+Please make sure your reading and commenting on enhancements as they come in.
+
+4.  Maintaining the overall health of the repo according to the standards below.
+
+Please help make sure that we're following the practices below.  If you see something that's not quite right, step up and fix it.
+
+5.  Helping with backports
+
+As part of our release process we initially merge into Main and then make conscious decisions about where and how to backport features.  As a maintainer, you need to be looking for fixes and features that need to be backported and then doing the backport.
+
+6.  Running releases
+
+Once we get into a regular cadence of release (past 1.0.0) maintainers should expect to act as release managers (see below).
+
+## Versions
+
+OpenSearch uses [semantic versioning aka semver](https://semver.org/).
+
+Here's a quick primer from [Apache’s versioning doc](https://commons.apache.org/releases/versioning.html) :
+
+A release number is comprised of 3 components: the major release number, the minor release number, and an optional point release number. Here is a sample release number:
+2.0.4
+and it can be broken into three parts:
+
+    * major release: 2
+    * minor release: 0
+    * point release: 4
+
+The next release of this component would increment the appropriate part of the release number, depending on the type of release (major, minor, or point). For example, a subsequent minor release would be version 2.1, or a subsequent major release would be 3.0.
+Note that release numbers are composed of three _integers_, not three digits. Hence if the current release is 3.9.4, the next minor release is 3.10.0.
+
+OpenSearch and OpenSearch Dashboards will release major version together.   They will NOT synchronize minor release — whenever the team feels they’re ready to release a minor version or patch (modulo the schedule above), they should release.
+
+What we guarantee is that any major release of Dashboards is compatible with the same major release of OpenSearch.  For example: 3.2.1 of Dashboards will work with 3.0.4 of OpenSearch, but 2.3.1 of Dashboards is not guaranteed to work with 3.0.4 of OpenSearch
```

Comment:
What about the opposite? 
Will an OpenSearch-Dashboards v3.2.1 be compatible with OpenSearch v2.0.1?